createLoadableVisibilityComponent.js

```diff
@@ -16,7 +16,7 @@ function _extends() { _extends = Object.assign || function (target) { for (var i
 var trackedElements = new Map();
 var options = {
   threshold: 0,
-  rootMargin: "0px 0px 100% 0px"
+  rootMargin: "0px 0px 500px 0px"
```

Comment:
Why are we changing this to an absolute value?

Reply:
This was actually part of src/createLoadableVisibilityComponent.js file earlier on v4.0.0 tag but looks like it was not part of build. So, post running the build command this got updated.